validator/specs/lib/validateTilesetSpec.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var validateTileset = require('../../lib/validateTileset');
+var clone = require('clone');
+
+describe('validateTileset', function() {
+
+    it('reads a valid JSON', function(done) {
+        var tileset = {
```

Comment:
Place this variable outside the `it` because it will likely be used by multiple tests.